src/General/Typography/Title.tsx

```diff
@@ -35,6 +37,7 @@ interface Props {
   color?: string;
   ellipsis?: boolean;
   tag?: tagType;
+  upperCase?: boolean;
```

Comment:
uppercase is a single word not upper and case right?

Reply:
Yes, I don't know why did I beak it into two!